home-assistant-plugin/custom_components/xknx/climate.py

```diff
@@ -177,6 +183,7 @@ def async_add_entities_config(hass, config, async_add_entities):
         group_address_setpoint_shift_state=config.get(
             CONF_SETPOINT_SHIFT_STATE_ADDRESS
         ),
+        setpoint_shift_mode=config.get(CONF_SETPOINT_SHIFT_MODE),
```

Comment:
Required and default values shall be accessed by `config[]` instead of `config.get()`